agent/base

```diff
@@ -230,6 +230,17 @@ function generate_inventory {
 	ls $tools_group_dir | awk -F@ '/^remote/ {print $2};'
 }
 
+# generate inventory file with controller and remotes for stockpile
```

Comment:
This looks like a repeat of an existing function.  Any way we can parameterize this and collapse into one?

Reply:
Fixed.